src/db/admin.py

```diff
@@ -0,0 +1,21 @@
+class Admin:
+	def __init__(self, db_conn):
+		self.db_conn = db_conn
+		self.interface_name = 'admin'
+
+	def get_semester_default(self):
+		return self.db_conn.execute("""
+			select
+				semester
+			from
+				admin_setting
+		""", None, True)
+
+	def set_semester_default(self, sem):
```

Comment:
`sem` -> `semester`, we have plenty horizontal space, for the rest of the codebase lets aim to be as explicit as possible in naming, easier to reason about.